templates/helpers.rb

```diff
@@ -314,6 +316,47 @@ def in_context name
     @convert_context.pop
     result
   end
+
+  STEM_EQNUMS_AMS = 'ams'
+  STEM_EQNUMS_NONE = 'none'
+  STEM_EQNUMS_VALID_VALUES = [
+    STEM_EQNUMS_NONE,
+    STEM_EQNUMS_AMS,
+    'all'
+  ]
+
+  # Generate the Mathjax markup to process STEM expressions
+  # @param cdn_base [String]
+  # @return [String]
+  def generate_stem(cdn_base)
+    if attr?(:stem)
+      eqnums_val = attr('eqnums', STEM_EQNUMS_NONE)
```

Comment:
I think we need to call `#downcase` to make sure that `:eqnums: ALL` will be correctly recognized.

Reply:
Done